web/sendgrid_webhooks.py

```
@@ -0,0 +1,183 @@
+from django.contrib.auth.models import User
+from django.http import HttpResponse
+from django.utils import timezone
+from django.views.decorators.csrf import csrf_exempt
+
+from .models import EmailEvent
+from .slack import send_slack_notification
+
+logger = logging.getLogger(__name__)
+
+
+from django.http import HttpResponse, HttpResponseForbidden
```

Ruff (0.8.2)

Module level import not at top of file

(E402)